src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -280,7 +285,11 @@ export const ConfirmSummary = ({
           component={RecipientAddress}
         />
         <WithLazyMount
-          shouldUse={!walletSupportsBuyAssetChain || disableThorNativeSmartContractReceive === true}
+          shouldUse={
+            Boolean(
+              !walletSupportsBuyAssetChain || disableThorNativeSmartContractReceive === true,
+            ) && !isAccountLoading
```

Comment:
preferably-blocking: with the current implementation, this will *not*  display the manual address entry until *all* accounts have loaded.

It may take some significant time before *all* accounts are loaded, and users will be stuck in "Accounts Loading" state in the meantime, despite actually having an AccountId for the requested chain in their portfolio.
  suggestion: check if there is an account for chain first, and only fallback to accounts loading checks otherwise vs. consistently using it as a guard
  
```diff
diff --git a/src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx b/src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx
index 26cdefd39a..e04f616d13 100644
--- a/src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx
+++ b/src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx
@@ -117,6 +117,14 @@ export const ConfirmSummary = ({
     buyAsset.chainId,
     buyAsset.assetId,
   ])
+
+  const displayManualAddressEntry = useMemo(() => {
+    // If accounts have finished loading, *and* there is no runtime chain support for the buy asset, display manual address entry
+    if (!walletSupportsBuyAssetChain) return !isAccountLoading
+
+    return disableThorNativeSmartContractReceive === true
+  }, [walletSupportsBuyAssetChain, disableThorNativeSmartContractReceive, isAccountLoading])
+
   const quoteHasError = useMemo(() => {
     if (!isAnyTradeQuoteLoaded) return false
     return !!activeQuoteErrors?.length || !!quoteRequestErrors?.length
@@ -285,11 +293,7 @@ export const ConfirmSummary = ({
           component={RecipientAddress}
         />
         <WithLazyMount
-          shouldUse={
-            Boolean(
-              !walletSupportsBuyAssetChain || disableThorNativeSmartContractReceive === true,
-            ) && !isAccountLoading
-          }
+          shouldUse={displayManualAddressEntry}
           shouldForceManualAddressEntry={disableThorNativeSmartContractReceive}
           component={ManualAddressEntry}
           description={manualAddressEntryDescription}
```

Reply:
Cool! Why not, I did apply the diff in [2d57d12](https://github.com/shapeshift/web/pull/7629/commits/2d57d12de291dbebf2c14ad97f3338aff5d7b571)